### PR TITLE
🍿 Add translation popup component

### LINKED ISSUE #435: Add popover for automated translations
image
We want to add this [popover](https://ant.design/components/popover/) saying if a translation is automated. For now, I don't have the backend completely set up for this, so we can just create the UI without connecting it to anything.
[icon docs](https://ant.design/components/icon/)

(Question for you to ask Lam/think about when the time comes - when clicking "Report an error", should a [message](https://ant.design/components/message/) pop up at the top of the screen when the report has been sent?)

### PR DESCRIPTION
**Pull request by Feeshay**
## Status:
:rocket: Ready

## Description
* Add component to display warning about automated translations
* Move error reporting endpoint and link to frontend

Fixes #435

## Todos

- [ ] Do we want this component anywhere else? (currently only in resource detail page)
- [ ] Add conditional rendering to display this only when there's an unverified translation. Also need to add endpoints/update backend model to see if the translation is verified or not.

## Screenshots
image
Mobile:
image

### CODE REVIEW

File: client/src/components/TranslationPopup.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      message.success('Thank you for your feedback!');
+    } else {
+      message.error('There was a problem submitting a report.');
+    }
+  };
+
+  return (
+    <Tooltip
+      color="white"
+      placement={isMobile ? 'top' : 'right'}
```
Comment (jackieo5023) [inline client/src/components/TranslationPopup.jsx:31]: nice

File: client/src/utils/api.js [hunk 2]
```diff
@@ -410,3 +410,18 @@ export const editResourceCategories = (
       },
     );
 };
+
+export const reportTranslationError = (
+  id: string,
+  language: string,
+  type: string,
+): ApiResponse<void> => {
```
Comment (jackieo5023) [inline client/src/utils/api.js:418]: Do you wanna make a flow type for the response and add it here instead of void?

**Review by jackieo5023 — APPROVED** [review]
Looks great!! Thanks Michael :)

File: client/src/utils/api.js (hunk 2 above)
Comment (Feeshay) [reply client/src/utils/api.js]: ye

**Pull request merged**